app/models/profile.rb

```diff
@@ -1,12 +1,16 @@
 class Profile < ActiveRecord::Base
-  belongs_to :user
+  belongs_to :user, dependent: :destroy
   has_many :memberships
   has_many :organisations, through: :memberships
 
+  accepts_nested_attributes_for :user
+
   validates :name, :address, :postcode, :email, :tel, :mobile, presence: true
 
   validates :user_id, uniqueness: true, allow_nil: true
 
+  validates :email, uniqueness: true
```

Comment:
Does not validate that the tel and mobile fields cannot contain characters. Allow only 0-9, spaces, plus, brackets?